go.mod

```diff
@@ -11,13 +11,23 @@ require (
 	github.com/btcsuite/btcutil v1.0.3-0.20201208143702-a53e38424cce
 	github.com/fortytw2/leaktest v1.3.0
 	github.com/go-kit/kit v0.10.0
+	github.com/go-lintpack/lintpack v0.5.2 // indirect
 	github.com/gogo/protobuf v1.3.2
 	github.com/golang/protobuf v1.5.2
+	github.com/golangci/errcheck v0.0.0-20181223084120-ef45e06d44b6 // indirect
+	github.com/golangci/go-tools v0.0.0-20190318055746-e32c54105b7c // indirect
+	github.com/golangci/goconst v0.0.0-20180610141641-041c5f2b40f3 // indirect
+	github.com/golangci/gocyclo v0.0.0-20180528134321-2becd97e67ee // indirect
+	github.com/golangci/golangci-lint v1.38.0 // indirect
```

Comment:
I feel like this (now) isn't indirect, and I think the ideal situation is where we don't have the indirect dependencies in the go.mod file, but I don't think it really matters.

Reply:
This won't end up in our users' `go.mod` so I think it should be fine?

Reply:
IMO is this OK 👍